### ISSUE TITLE
error in rinning code

### ISSUE THREAD

**Issue by xfeng853** [craigleili/3DLocalMultiViewDesc#6]
the error is  "  UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.  "  
my torch version is the same as the "requirements" .  when i reduce torch version, another error is appeared, "no module named torch.utils.tensorbard" ,

**Comment by craigleili** [comment]
Hi,

Have you checked if the versions of `pytorch-ignite` and `torch` match? And it seems the UserWarning is not a severe issue - you could probably just ignore it.

All the best

**Comment by xfeng853** [comment]
Thank for your help.  And i  encountered another problem. When I processed the dataset, I did not generate the file  *_ keypts. npy .   I just have cloud_bin_0.frames.pkl , cloud_bin_0.ply , cloud_bin_0.pose.npy , cloud_bin_0.radius.npy , and the fold log_kpts is empty, I don't know if this is normal?

**Comment by xfeng853** [comment]
Thanks for your help.  And i have another question , i don‘t know how to generate *keypts.npy  , and when I finish processing the data，the fold log_kpts is empty.  Is this normal?

**Comment by craigleili** [comment]
The `*.keypts.npy` files are used for testing and have been included in the processed benchmark, which can be downloaded [here](https://github.com/craigleili/3DLocalMultiViewDesc#evaluation).

**Comment by xfeng853** [comment]
Thanks for your reply. I now want to generate a corresponding descriptor based on my data. I read this paper you wrote and think I only need to give a point of interest to generate a multi-view descriptor, but I don't know how to generate it now. I read the readme file, there is only one corresponding command,  “python main_mvdesc.py test configs/ours_3dmatch.yaml ” , but the *.keypts.npy  files are needed, and you said this is for testing. I just want to get the descriptors now , What should I do?

**Comment by craigleili** [comment]
Hi,

Each `*.keypts.npy` file is a 1-D integer array of keypoint indices, which you can specify for your data. You may refer to [this data loader](https://github.com/craigleili/3DLocalMultiViewDesc/blob/eb7d6912886da851e673e6a8528e9ba04001d73a/data/pointclouds.py?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L239-L240)

All the best
